src/SwedbankPay.Sdk.Tests/TestBuilders/PaymentRequestBuilder.cs

```diff
@@ -224,5 +239,27 @@
             };
             return this;
         }

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5814474)